torch/csrc/autograd/python_variable.cpp

```diff
@@ -682,6 +723,7 @@ static struct PyGetSetDef THPVariable_properties[] = {
   {"data", (getter)THPVariable_get_data, (setter)THPVariable_set_data, nullptr, nullptr},
   {"_grad", (getter)THPVariable_get_grad, (setter)THPVariable_set_grad, nullptr, nullptr}, // Allows the python class to override .grad
   {"grad", (getter)THPVariable_get_grad, (setter)THPVariable_set_grad, nullptr, nullptr},
+  {"_fw_grad", (getter)THPVariable_get_fw_grad, (setter)THPVariable_set_fw_grad, nullptr, nullptr},
```

Comment:
The final API for this will be a `fw_grad` field and a `set_fw_grad(value)` function.
The first one can be used to access, set and reset the value for the forward grad.
Unfortunately, because of the constraints on `[Forward Grad Layout]`, the Tensor that gets set might not be the one that is given.
So setting using the attribute will only be allowed when the given Tensor can actually be used. If this should be an error or a warning will be decided later.
Otherwise, you can use the `set_fw_grad()` function that will return the Tensor that was actually set.